### ISSUE TITLE
[WoW Classic Era]Error Loading the MicroBar module

### ISSUE THREAD

**Issue by robisonkarls** [Merathilis/ElvUI_MerathilisUI#60]
### What is the issue you are having?

I get the following errors when loggin in WoW Classic Era

![image](https://user-images.githubusercontent.com/167655/201712050-6b86ebfe-9270-40df-9fc3-15d46811b2c1.png)
![image](https://user-images.githubusercontent.com/167655/201712105-48cdcf45-d1ea-4732-bf0d-3aa56e8b019f.png)
![image](https://user-images.githubusercontent.com/167655/201712168-b7a0d30c-23ad-4561-a90b-9d247ded093d.png)

### What is the expected behavior?

The expected behaviour  is to have the proper microbar loaded

### What actually happened?

![image](https://user-images.githubusercontent.com/167655/201712050-6b86ebfe-9270-40df-9fc3-15d46811b2c1.png)
![image](https://user-images.githubusercontent.com/167655/201712105-48cdcf45-d1ea-4732-bf0d-3aa56e8b019f.png)
![image](https://user-images.githubusercontent.com/167655/201712168-b7a0d30c-23ad-4561-a90b-9d247ded093d.png)

### Errors

```shell
![image](https://user-images.githubusercontent.com/167655/201712050-6b86ebfe-9270-40df-9fc3-15d46811b2c1.png)
![image](https://user-images.githubusercontent.com/167655/201712105-48cdcf45-d1ea-4732-bf0d-3aa56e8b019f.png)
![image](https://user-images.githubusercontent.com/167655/201712168-b7a0d30c-23ad-4561-a90b-9d247ded093d.png)
```

### ElvUI Changelog/Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://github.com/Merathilis/ElvUI_MerathilisUI/blob/development/CHANGELOG.md) and done the [Troubleshooting Steps](https://github.com/Merathilis/ElvUI_MerathilisUI/wiki/Troubleshooting).
- [ ] I have disabled all addons except the AddOns called 'ElvUI' and 'ElvUI OptionsUI' and 'ElvUI_MerathilisUI' by using `/muidebug on`? This is important in order to rule out that any of your other addons are causing the issue. If you manage to pinpoint a conflict between ElvUI and/or MerathilisUI and another addon, make sure you include that information.
- [X] I have verfied my issue doesn't exist in the default UI.
- [X] I gave you information above on which XPack i play Classic, WotLK or Retail.

**Comment by khori0** [comment]
Turning off Additional Text under elvui>mui>modules>micro bar>Display gets rid of the error

**Comment by Merathilis** [comment]
this should be fixed with 5.40
